### DISCUSSION TITLE
Create single HTML report

### DISCUSSION THREAD

**Discussion by falukky** [MihanEntalpo/allure-single-html-file#23] [Q&A]
So in my project I have the report folder where all .json files created and after finish my test I am using this command:

`allure-combine ./reports` 

And got this error:
 ./reports\index.html doesn't exists, but it should!

My index.html file located in different folder :
C:\Users\myuser\AppData\Local\Temp\5676934553641883404\allure-report

So I try to change directory inside this folder and then:
`allure-combine PATH_TO_MY_REPORTS_FOLDER`

But this not helps

**EDIT**

Finally I was able to understand how its works:.
After the tests finished and I have all .json files and after I created allure report  using the command `allure serve ./reports` only after that the tamp folder with all the files (index.html etc) created so I copied all the content of this folder into my reports folder and then the command `allure-combine ./reports` works and the report files created so now I wonder how to automatically generate this report so I can sent it via email because I need to know first the location of the temp folder and also run the command `allure serve ./reports` to generate this temp folder...

Any suggestions ?

**Comment by MihanEntalpo** [comment]
Hello. Did you try to run `allure report ./reports` ? It should create index.html file inside of ./reports folder. 
If it doesn't do this, I'd try to run `allure server --help` to get supported command line attributes, maybe there are some parameter to specify output folder to be used instead of temporary one.

**Comment by adarshpv9746** [comment]
When I try to run the script

`"generate-allure-standalone": "allure generate ./reports/json/allure-results --clean && node ./node_modules/allure-single-html-file-js/combine.js ./allure-report && node ./allure-config/rename-move.js && node ./allure-config/htmltopdf.js"`,

Recently it started showing the error 
`Error: ENOENT: no such file or directory, open './allure-report/https://www.googletagmanager.com/gtag/js?id=G-FVWC4GKEYS'`

this used to work properly
help me with this issue

**Comment by MihanEntalpo** [comment]
Hello
In your command line 
```
allure generate ./reports/json/allure-results --clean && node ./node_modules/allure-single-html-file-js/combine.js ./allure-report && node ./allure-config/rename-move.js && node ./allure-config/htmltopdf.js
```
I don't see any call for my allure-combine python script, you are running some node.js module that is failing.

I've googled and found this module that is the port of my script to JavaScript https://socket.dev/npm/package/allure-single-html-file-js and it seems that you have problems with that.

So, as I'm not the author of that port, I suggest you to create issue in the port's repository, which is located here: https://github.com/aruiz-caritsqa/allure-single-html-file-js

Good luck :)
